js/client-topbar.js

```diff
@@ -1010,10 +1010,21 @@
 				buf += '<div id="gapi-custom-signin" style="width:240px;margin:0 auto">[loading Google log-in button]</div>';
 				buf += '<p class="buttonbar"><button name="close">Cancel</button></p>';
 			} else {
-				buf += '<p><label class="label">Password: <input class="textbox autofocus" type="password" name="password" autocomplete="current-password"></label></p>';
+				buf += '<p><label class="label">Password: <input class="textbox autofocus" type="password" name="password" id="pwd" autocomplete="current-password"></label><i class="fa fa-eye-slash showpassword" onclick="showPassword(this)"></i></p>';
```

Comment:
Putting an `onclick` handler on an `<i>` tag is bad for accessibility, btw.

Reply:
You'd want to wrap it in a `<button>` tag so screenreaders and keyboard users know it's clickable.